checkov/sca_package_2/output.py

```diff
@@ -141,10 +145,22 @@ def create_cli_output(fixable: bool = True, *cve_records: list[Record]) -> str:
                             parsed_version = packaging_version.parse(root_package_fix_version.strip())
                             fix_versions_lists.append([parsed_version])
 
-                    root_package_lines = validate_lines(record.vulnerability_details.get("root_package_file_line_range"))
+                    root_package_lines = validate_lines(
+                        record.vulnerability_details.get("root_package_file_line_range"))
                     if lines or root_package_lines:
                         lines_details_found_cves = True
 
+                    if record.vulnerability_details and record.vulnerability_details.get("risk_factors", {}).get(
+                            "IsUsed"):
+                        cve_count.used += 1
+
+                    if record.vulnerability_details is not None:
+                        reachability_risk_factors_tmp = {key: value for key, value in
+                                                         record.vulnerability_details.get("risk_factors", {}).items()
+                                                         if key in InterestingRiskFactors}
+                    else:
+                        reachability_risk_factors_tmp = {}
```

Comment:
This part is redundant as if none of the dictionary keys is in `InterestingRiskFactors` it will default to an empty dictionary as you explicitly defined.

Reply:
But if the condition is false reachability_risk_factors_tmp will not be defined and when calling it we will have NameError, right? @Eliran-Turgeman